docker-compose.yml

```diff
@@ -3,8 +3,11 @@ version: '3.8'
 services:
   mailserver:
     image: docker.io/mailserver/docker-mailserver:latest
-    hostname: <HOSTNAME> # <-- CHANGE THIS
-    domainname: <DOMAINNAME> # <-- CHANGE THIS
+    # build:
+    #   context: .
+    #   dockerfile: Dockerfile
```

Comment:
Do we want this in the docker-compose.yml?

Reply:
Forgot to remove it 😅  and missed it with the review. I've removed it now